scripts/missions/toau/07_Westerly_Winds.lua

```diff
@@ -60,6 +60,7 @@ mission.sections =
 
                 [3028] = function(player, csid, option, npc)
                     if mission:complete(player) then
+                        player:setCharVar('WesterlyWinds_date', getMidnight())
```

Comment:
This awesome! 
However the GMs have been requesting that we keep variable names similar from quest to quest and mission to mission.
So here we should use one of the 3 most common variable names: "Stage", "Option", "Prog".  "Stage" is most commonly used for time waits.
```suggestion
                        mission:setVar(player, 'Stage', getMidnight())
```

It's not something we've done historically, but we've been pushing to keep these variable names consistent since launching Horizon

Reply:
how do I access that variable from the next mission?

Reply:
Ah! So it's for the next mission, eh? So the way quest or mission:setVar(player, 'Prog') sets the variable, is it sets a char var like the following:  player:setCharVar("Mission[0][12]Prog", 2) for example. (Case sensitive)

Where Mission can either be Mission or Quest, the first var, [0] is the mission line ID or quest zone ID, the second var, [12], is the mission or quest ID. And finally the Prog is the variable name itself.

I hope that makes sense! So you would go, player:setCharVar("Mission[4][6]Stage", getMidnight())

You'll be able to see a full list of these IDs in both quests.lua and missions.lua